docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -346,33 +393,170 @@ txn_type      | debit
+
+```sql
+SET force_global_transaction = FALSE;
+DELETE FROM bank_transactions WHERE user_id = 200 AND account_id = 10005;
+```
+
+```output
+ERROR:  Illegal state: Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64: . Errors from tablet servers: [Illegal state (yb/client/transaction.cc:284): Nonlocal tablet accessed in local transaction: tablet c5a611afd571455e80450bd553a24a64]
+```
+
+We are attempting to delete from the main table (`bank_transactions` rather than `bank_transactions_eu_west_1`) and not specifying the partition column (there's no `geo_partition = 'EU'` clause). This means that YugabyteDB is unable to tell that the row being deleted is in fact located in eu-central-1. To fix this, we could instead run:
+
+```sql
+DELETE FROM bank_transactions_eu_west_1 WHERE user_id = 200 AND account_id = 10005;
+```
+
+```output
+DELETE 1
+```
+
+## Step 6. Adding a new geographic location
 
 Assume that after a while, our fictitious Yuga Bank gets a lot of customers across the globe, and wants to offer the service to residents of Brazil, which also has data residency laws. Thanks to row-level geo-partitioning, this can be accomplished easily. We can simply add a new partition and pin it to the AWS South America (São Paulo) region `sa-east-1` as shown below.
 
+First, create the tablespace:
+
 ```sql
 CREATE TABLESPACE sa_east_1_tablespace WITH (
     replica_placement='{"num_replicas": 3, "placement_blocks":
       [{"cloud":"aws","region":"sa-east-1","zone":"sa-east-1a","min_num_replicas":1},
       {"cloud":"aws","region":"sa-east-1","zone":"sa-east-1b","min_num_replicas":1},
       {"cloud":"aws","region":"sa-east-1","zone":"sa-east-1c","min_num_replicas":1}]}'
     );
+```
+
+Next, create the transaction table and adjust the placement. (Replace the IP addresses with those of your YB-Master servers.)
 
-CREATE TABLE transactions_brazil
-    PARTITION OF transactions
+```sh
+./bin/yb-admin \
+    -master_addresses 127.0.0.1:7100,127.0.0.4:7100,127.0.0.7:7100 \
+    create_transaction_table transactions_sa_east_1
+```
+```sh
+./bin/yb-admin \
+    -master_addresses 127.0.0.1:7100,127.0.0.4:7100,127.0.0.7:7100 \
+    modify_table_placement_info system transactions_sa_east_1 \
+    aws.sa-east-1.sa-east-1a,aws.sa-east-1.sa-east-1b,aws.sa-east-1.sa-east-1c 3
+```
+
+Finally, create the partition for Brazil:
+
+```sql
+CREATE TABLE bank_transactions_brazil
+    PARTITION OF bank_transactions
       (user_id, account_id, geo_partition, account_type,
        amount, txn_type, created_at,
        PRIMARY KEY (user_id HASH, account_id, geo_partition))
     FOR VALUES IN ('Brazil') TABLESPACE sa_east_1_tablespace;
 ```
 
-And with that, the new region is ready to store transactions of the residents of Brazil.
+And with that, the new region is ready to store bank transactions of the residents of Brazil.
```

Comment:
Connect to brazil endpoint and insert the rows.

Reply:
The example here is actually a fast path insert analogous to those in step 3; it's possible to change it to be analogous to those in step 5 instead, but that would probably just complicate the example without introducing much value.